remote-monitoring/docker-compose.dotnet.yml

```diff
@@ -24,6 +24,8 @@ services:
       - devicesimulation
       - telemetry
       - uiconfig
+    volumes:
+      - /app/webui-config.js:/app/build/config.js:ro
```

Comment:
Can we keep this as environment variables? It will be simpler to do that for enterprise version as well.

Reply:
discussed offline, this way of injecting env vars it to avoid rebuilding the node.js app every time the webui container starts